scan/lib/scan/runner.rb

```diff
@@ -98,7 +98,7 @@ def copy_simulator_logs
       UI.header("Collecting system logs")
       Scan.devices.each do |device|
         logarchive_dest = File.join(Scan.config[:output_directory], "system_logs-#{device.name}_#{device.os_type}_#{device.os_version}.logarchive")
```

Comment:
I think I'll change this to just a folder name that has the device details (without system_logs-...etc)